core/aclmgmt/resourceprovider_test.go

```diff
@@ -57,17 +57,32 @@ type signerSerializer interface {
 //go:generate counterfeiter -o mocks/defaultaclprovider.go --fake-name DefaultACLProvider . defaultACLProvider
 
 func TestPolicyBase(t *testing.T) {
-	peval := &mockPolicyEvaluatorImpl{pmap: map[string]string{"res": "pol"}, peval: map[string]error{"pol": nil}}
-	pprov := newPolicyProvider(peval)
-	sProp, _ := protoutil.MockSignedEndorserProposalOrPanic("A", &peer.ChaincodeSpec{}, []byte("Alice"), []byte("msg1"))
-	err := pprov.CheckACL("pol", sProp)
-	require.NoError(t, err)
-
-	signer := &mocks.SignerSerializer{}
-	env, err := protoutil.CreateSignedEnvelope(common.HeaderType_CONFIG, "myc", signer, &common.ConfigEnvelope{}, 0, 0)
-	require.NoError(t, err)
-	err = pprov.CheckACL("pol", env)
-	require.NoError(t, err)
+	evaluator := &mockPolicyEvaluatorImpl{pmap: map[string]string{"res": "pol"}, peval: map[string]error{"pol": nil}}
+	provider := newPolicyProvider(evaluator)
+
+	t.Run("SignedEndorserProposal", func(t *testing.T) {
+		sProp, _ := protoutil.MockSignedEndorserProposalOrPanic("A", &peer.ChaincodeSpec{}, []byte("Alice"), []byte("msg1"))
+		err := provider.CheckACL("pol", sProp)
+		require.NoError(t, err)
+	})
+
+	t.Run("SignedEndorserProposal", func(t *testing.T) {
```

Comment:
Might want to use different names -- `SignedProposal` and `SignedEnvelope`?

Reply:
Good spot! Bad cut & paste. I was aiming to just use the exact names of the types being tested in the description.